docs/features/logforwarding/outputs/otlp-forwarding.adoc

```diff
@@ -49,98 +49,93 @@ spec:
-
-https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L812[Tuning Specification] for _observability.openshift.io/v1_
+.API Details for _observability.openshift.io/v1_
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L261[TLS Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L242[HTTP Auth Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L201[HTTP Tuning Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L1225[OTLP Tuning Spec]
 
 
 == Data Model
 
 === Semantic Convention
-The Semantic Conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry as *Attributes*.
+The Semantic Conventions in OpenTelemetry (OTel) define a *Resource* as an immutable representation of the entity producing telemetry as *Attributes*.
+
+All *Attributes* are included in the *Resource* object. For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
 
-For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
+The grouping and reducing of common resource attributes is a key aspect of sending logs via OpenTelemetry.
 
-All of these *Attributes* are included in the *Resource* object.  This grouping and reducing of common attributes is a key aspect of sending logs via OpenTelemetry.
+.Cluster Logging Reference Docs
+* https://github.com/rhobs/observability-data-model/blob/main/cluster-logging.md[OTel Data Model Draft]
+* https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-otel-support.md[OTel Enhancement Doc]
+* https://github.com/openshift/cluster-logging-operator/blob/master/docs/reference/datamodels/viaq/v1.adoc[ViaQ Data Model]
 
-.OTEL References
+.OTel References
+* https://opentelemetry.io/docs/specs/otlp/[OTLP Specification]
 * https://opentelemetry.io/docs/specs/semconv/[Semantic Conventions]
-* https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+* https://opentelemetry.io/docs/specs/otel/logs/data-model/[OTel Logs Data Model]
 * https://opentelemetry.io/docs/specs/semconv/general/logs/[General Logs Attributes]
 
 === Data Mapping
-This document is not a complete spec and is mainly used as a guide.
+OpenShift Logging uses Vector Remap Language https://vector.dev/docs/reference/vrl/[(VRL)] to map and enrich the logging data.  The existing ViaQ fields are mapped to follow the semantic conventions of OpenTelemetry
```

Comment:
Consider removing any language regarding the implementation (i.e. vector, vrl, etc); It's not relevant

Reply:
removed this entire section